crypto/src/action/error.rs

```diff
@@ -0,0 +1,27 @@
+use thiserror;
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
```

Comment:
Maybe this could be `ProtoError` or something, so that it's scoped to errors that can occur while deserializing?

Reply:
superseded by https://github.com/penumbra-zone/penumbra/pull/86#issuecomment-947255729